frontend/pages/faq.tsx

```diff
@@ -0,0 +1,55 @@
+import { GetStaticProps } from "next"
+import { useTranslation } from "next-i18next"
+import { serverSideTranslations } from "next-i18next/serverSideTranslations"
+import { NextSeo } from "next-seo"
+import Link from "next/link"
+
+const Faq = (): JSX.Element => {
+  const { t } = useTranslation()
+
+  return (
+    <>
+      <NextSeo title={t("faq")} description={t("faq-description")} />
+      <div className="flex flex-col px-[5%] text-justify md:px-[20%] 2xl:px-[30%]">
+        <h1>Frequently Asked Questions</h1>
+
+        <h2>Verified Apps</h2>
+
+        <h3>What does the checkmark under an app name mean?</h3>
+        <p>
+          Some apps have a checkmark on the app page under the developer name.
+          This means the app is published on Flathub by its original developer
+          or a third party approved by the developer. Some apps are published by
```

Comment:
I think this reads hard/misleading. A break/paragraph after the sentence ended might help? 
```suggestion
          or a third party approved by the developer.</p><p> Some apps are published by
```